modules/custom/openy_search/SMOKE_TESTS.md

```diff
@@ -0,0 +1,81 @@
+# Welcome to Grid Content smoke tests documentation
+
+In order for Open Y Grid Content being tested in a short timeframe, please follow steps below
```

Comment:
Same here - `Open Y Grid Content`, should be `Open Y Search `

Reply:
addressed https://github.com/ymcatwincities/openy/pull/2535/commits/f53eaddc5655b0974cf125f8d32468d4bb7788a4